eve-api-data/src/test/resources/application-context.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Java configuration has become the norm.  While spring boot will still listen to configuration xml's its become preferred to use java config.  This file should be unnecessary.